magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -85,27 +85,36 @@ export default class DatasetPageSuggestForm extends React.Component {
         };
         const formProps = {
             title: false,
+            description:
+                "Your question will be sent to the publishers of the data: " +
+                this.props.contactPoint,
             namePlaceHolder: "Dorothy Hill",
             emailPlaceHolder: "dorothyhill@example.com",
             textAreaPlaceHolder:
                 "Ask a question or report a problem about this dataset.",
             textAreaLabel: "What would you like to ask about this dataset?",
-            successHeader: "Your request has been submitted!"
+            successHeader:
+                "Your request has been submitted to: " +
+                this.props.contactPoint +
+                "!"
         };
 
         return (
             <React.Fragment>
                 {/* If the form is posted don't show the text in the below para*/}
-                {!this.state.showSuggest && (
-                    <div className="dataset-button-container no-print">
-                        <button
-                            className="au-btn au-btn--secondary ask-question-button"
-                            onClick={this.toggleShowForm}
-                        >
-                            Ask a question about this dataset
-                        </button>
-                    </div>
-                )}
+                {!this.state.showSuggest &&
+                    (this.props.contactPoint ? (
+                        <div className="dataset-button-container no-print">
+                            <button
+                                className="au-btn au-btn--secondary ask-question-button"
+                                onClick={this.toggleShowForm}
+                            >
+                                Ask a question about this dataset
+                            </button>
+                        </div>
+                    ) : (
+                        <React.Fragment />
```

Comment:
This can be a `null` instead --- <React.Fragment /> will be an object of type fragment which will have to be created by react

Reply:
Done!